tsconfig.json

```diff
@@ -2,17 +2,16 @@
 	"compilerOptions": {
 		"strict": true,
 		"target": "es6",
-		"module": "commonjs",
-		"moduleResolution": "node",
+		"module": "Node16",
+		"moduleResolution": "Node16",
```

Comment:
We need this to support `exports` from `package.json` (https://github.com/typescript-eslint/typescript-eslint/issues/7284).